tests/test-a8c-files-utils.php

```diff
@@ -1,6 +1,48 @@
 <?php
 
 class VIP_Go_A8C_Files_Utils_Test extends WP_UnitTestCase {
+
+	private $a8c_files;
+
+	public static function setUpBeforeClass() {
+		parent::setUpBeforeClass();
+
+		require_once( __DIR__ . '/../a8c-files.php' );
+		require_once( __DIR__ . '/../files/class-api-client.php' );
+	}
+
+	public function setUp() {
+		parent::setUp();
+	}
+
+	public function tearDown() {
+		$this->api_client = null;
+		parent::tearDown();
+	}
+
+	private function setup_mock_a8c_files( $method_name, $expected ) {
```

Comment:
Minor: Based on the name alone, this seems like it's mocking `a8c_files`. Maybe `setup_a8c_files_and_mock_client` is a bit clearer (even though it's longer :))

Reply:
Yeas, I thought about that. Also the fact that in some of tests I'm mocking an object that I pass _another_ mock object into. :)